drivers/SmartThings/sonos/src/init.lua

```diff
@@ -1,24 +1,14 @@
 SONOS_API_KEY = require 'app_key'
 local old_log = require "log"
 
--- Forward all debug logs to hubcore info logs.
-old_log.debug = function(...)
-  old_log.info_with({hub_logs = true}, table.concat({"[DEBUG->INFO] ", ...}))
+-- Print all errors, warnings, and info to hubcore for now
+local log = {}
+log.warn = function(...)
```

Comment:
For Info/Error/Warn overrides to be global, we should override "`old_log`'s" functions after `require` but before re-scoping it to the `local log` identifier.